test/integration/auth.spec.ts

```diff
@@ -391,6 +391,11 @@ describe('admin.auth', () => {
         .then(() => {
           return admin.auth().verifySessionCookie(currentSessionCookie, true)
             .should.eventually.be.rejected.and.have.property('code', 'auth/session-cookie-revoked');
+        })
+        .then(() => {
+          // Wait an additional second to avoid token revocation for specified uid
+          // from affecting other tests that use the same uid.
+          return new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
I'm not entirely sure what this is expected to achieve. We are not using this UID in any other tests as far as I can see.  Plus we have a 1s delay before each revocation in the test suite. Can you please elaborate why this is needed?